storm-server/src/main/java/org/apache/storm/scheduler/resource/normalization/NormalizedResourceRequest.java

```diff
@@ -20,15 +20,15 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import net.minidev.json.JSONObject;
+import net.minidev.json.parser.JSONParser;
```

Comment:
Thanks for this pull request. I believe org.json.simple.parser.JSONParser was not always thread safe.